client/stylesheets/abstracts/_colors.scss

```diff
@@ -61,3 +65,6 @@ $button-border: darken($button, 20%);
 // Muriel
 $muriel-gray-dark-300: #969ca1;
```

Comment:
Just a side note that these can be removed/updated since variable names have been updated (depending on whether we merge this first or #2252).